### PR TITLE
creation of the flask project

### PR DESCRIPTION
**Pull request by rajitha1998**
## Changes proposed in the pull request
  * Creating a flask project
  * Generating the requirements.txt to record packages 
## Impact
  *  Overall Application

### CODE REVIEW

**Comment by rajitha1998** [comment]
Review required @janethavi @hasalfernando @nandulaperera @Purushothh

**Review by hasalfernando — APPROVED** [review]

**Review by hasalfernando — APPROVED** [review]

**Review by hasalfernando — APPROVED** [review]

**Review by janethavi — CHANGES_REQUESTED** [review]
@rajitha1998 Can you please add these files/folders(app.py,static/,templates/)  to a separate parent folder called "api"

**Comment by rajitha1998** [comment]
> @rajitha1998 Can you please add these files/folders(app.py,static/,templates/) to a separate parent folder called "api"

sure :)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+venv/
+api/venv/
+# IDEs folder
+.idea/**
+api/.idea/**
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+api/__pycache__/
+api/*.py[cod]
```
Comment (janethavi) [inline .gitignore]: @rajitha1998 Can you please add an EOF?

**Review by janethavi — CHANGES_REQUESTED** [review]

**Review by hasalfernando — APPROVED** [review]

**Review by janethavi — APPROVED** [review]

**Pull request merged**